FirebaseAuth/Sources/Public/FirebaseAuth/FIRUser.h

```diff
@@ -132,7 +132,9 @@ NS_SWIFT_NAME(User)
 
 /** @fn updateEmail:completion:
     @brief Updates the email address for the user. On success, the cached user profile data is
-        updated.
+   updated. Throws "auth/operation-not-allowed" error when [Email Enumeration
```

Comment:
Fix the error code. I think this is the web format of the error code.

Reply:
Yeah, thanks for catching this.